projects/fal_client/README.md

```diff
@@ -54,6 +54,18 @@ response = fal_client.run("fal-ai/whisper", arguments={"audio_url": audio_url})
 print(response["text"])
 ```
 
+## Encoding files as in-memory data URLs
+
+If you don't want to upload your file to our CDN service (for latency reasons, for example), you can encode it as a data URL and pass it directly to the client. Here's an example:
```

Comment:
this does not work for more than 10MB (someone reported 3MB but have not tested). We probably should add a coment

Reply:
we can probably add a check too.